compiler-tests/build.gradle.kts

```diff
@@ -5,7 +5,8 @@ plugins {
   java
 }
 
-val metroRuntimeClasspath: Configuration by configurations.creating
+val metroRuntimeClasspath: Configuration by configurations.creating { isTransitive = false }
+val anvilRuntimeClasspath: Configuration by configurations.creating { isTransitive = false }
```

Comment:
For my own understanding, why the `isTransitive` addition here? Just to have dependencies explicitly added?